vispy/gloo/gl/gl2.py

```diff
@@ -67,14 +75,18 @@ def _get_gl_func(name, restype, argtypes):
             fargs = (restype,) + argtypes
             ftype = ctypes.WINFUNCTYPE(*fargs)
             if not _have_get_proc_address:
-                raise RuntimeError('Function %s not available.' % name)
+                raise RuntimeError('Function %s not available '
+                                   '(OpenGL version %s).'
```

Comment:
Maybe `"OpenGL version is %s"` to make it clear this is the version in use, as opposed to the version required.